tests/test_config.py

```diff
@@ -91,3 +92,16 @@ def test_can_read_toml_rich_types(tmp_path):
 """
     )
     check_configuration_file_reader(config_file)
+
+
+def test_gets_correct_config_files(tmp_path):
```

Comment:
I cannot see where this test is called and how the `tmp_path` is set up. Are we sure this test is executed?

Reply:
`tmp_path` is a [pytest fixture](https://docs.pytest.org/en/stable/tmpdir.html#the-tmp-path-fixture)

Reply:
@Pierre-Sassoulas thanks for your answer. I should have seen that but i was a bit in a hurry.